mautic/mautic
# Search with wildcards

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | Fixes #8491



#### Description:

On few places search bar doesn't return correct results. This PR fixed it



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create company "Company test name"
3. Create focus "Focus test name"
4. Create email "Email test name"
5. Try in all pages (companies, focus, email) seach: test 
6. You find nothing
7. After PR you should return results as expected
